packages/hoppscotch-backend/src/admin/admin.model.ts

```diff
@@ -1,4 +1,5 @@
 import { ObjectType } from '@nestjs/graphql';
+import { User } from 'src/user/user.model';
 
 @ObjectType()
-export class Admin {}
+export class Admin extends User {}
```

Comment:
```suggestion
export class Admin {}
```
extending `User` results in errors when trying to run queries to fetch user-settings, user-environments etc.

Reply:
<img width="1512" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/14091627/aeca691b-011d-435c-be60-368dc0c8de49">

Reply:
```suggestion
export class Admin extends PartialType(User) {}
```
@balub, I'm planning to use `PartialType`.  

After using `PartialType`, from `admin` query:
There will be only 
> `uid, displayName, email, photoURL, isAdmin, createdOn, currentRESTSession, currentGQLSession`. 

And there won't be 
> `settings, environments, globalEnvironments, RESTHistory, GQLHistory`

![Screenshot 2023-10-18 at 5 40 19 PM](https://github.com/hoppscotch/hoppscotch/assets/30154643/ad8e716f-c090-4dd7-b9e9-ef5f2f7275e7)

I think, `settings, environments, etc` are SH-App-related fields instead of SH-Admin. 

Along with that, if in the future (assume), admin needs to know the `settings/environments/etc` of a particular user or himself (from admin dashboard), then, `infra.userInfo(uid)` is there to serve purpose.

Reply:
Okay. It makes sense.

Reply:
If this is the case we can remove `currentRESTSession` and `currentGQLSession` properties too.

Reply:
Okay, I'll omit that 3 fields.